packages/ui-components/src/components/dropdown/dropdown.types.ts

```diff
@@ -0,0 +1,22 @@
+import { EventEmitter } from '@stencil/core';
+import { EIconName, EOtherIconName } from '../icon/icon.types';
+import { EValidationState } from '../text-field/text-field.types';
+
+export const DROPDOWN_DEFAULT_PLACEHOLDER = 'Select an option';
```

Comment:
This could be on `packages/ui-components/src/components/dropdown/dropdown.config.ts`, right?